spyder/app/tour.py

```diff
@@ -982,13 +1044,22 @@ def _process_widgets(self, names, spy_window):
         dockwidgets = []
 
         for name in names:
-            base = name.split('.')[0]
-            temp = getattr(spy_window, base)
-
-            # Check if it is the current editor
-            if 'get_current_editor()' in name:
-                temp = temp.get_current_editor()
-                temp = getattr(temp, name.split('.')[-1])
+            try:
+                base = name.split('.')[0]
+                try:
+                    temp = getattr(spy_window, base)
+                except AttributeError:
+                    for item in spy_window.thirdparty_plugins:
+                        if type(item).__name__ == name:
+                            temp = item
+                            break
+                else:
+                    # Check if it is the current editor
+                    if 'get_current_editor()' in name:
+                        temp = temp.get_current_editor()
+                        temp = getattr(temp, name.split('.')[-1])

```

Comment:
```suggestion
                        temp = getattr(temp, name.split('.')[-1])
                    if temp is None:
                        raise
```

Re-raise the error so we get to the outer except block as we should